<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <buttongroups>
    <buttongroup name="nanCmapButtonGroup"/>
  </buttongroups>
  <class>Form</class>
  <widget class="QWidget" name="Form">
    <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
        <number>5</number>
      </property>
      <property name="topMargin">
        <number>5</number>
      </property>
      <property name="rightMargin">
        <number>5</number>
      </property>
      <property name="bottomMargin">
        <number>5</number>
      </property>
      <item row="0" column="1">
        <widget class="QTabWidget" name="tab_widget">
          <property name="currentIndex">
            <number>0</number>
          </property>
          <property name="elideMode">
            <enum>Qt::ElideNone</enum>
          </property>
          <!--============================= Size =============================-->
          <widget class="QWidget" name="size_tab">
            <attribute name="title">
              <string>Size</string>
            </attribute>
            <layout class="QGridLayout" name="verticalLayout">
              <property name="leftMargin">
                <number>4</number>
              </property>
              <property name="topMargin">
                <number>8</number>
              </property>
              <property name="rightMargin">
                <number>4</number>
              </property>
              <property name="bottomMargin">
                <number>4</number>
              </property>
              <property name="horizontalSpacing">
                <number>10</number>
              </property>
              <property name="verticalSpacing">
                <number>5</number>
              </property>
              <!--================================================================-->
              <item row="0" column="0">
                <widget class="QLabel" name="label_size_mode" alignment="Qt::AlignRight|Qt::AlignVCenter">
                  <property name="font">
                    <font>
                      <weight>50</weight>
                    </font>
                  </property>
                  <property name="text">
                    <string>Size mode:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QComboBox" name="combosel_size_mode">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="maximumSize">
                    <size>
                      <width>80</width>
                      <height>16777215</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item row="0" column="2">
                <widget class="QWidget" name="size_size">
                  <widget class="QLineEdit" name="valuetext_size"/>
                </widget>
              </item>
              <!--================================================================-->
              <item row="1" column="0" colspan="3">
                <widget class="QWidget" name="size_map_attributes">
                  <layout class="QGridLayout" name="size_map_grid">
                    <property name="leftMargin">
                      <number>0</number>
                    </property>
                    <property name="topMargin">
                      <number>0</number>
                    </property>
                    <property name="rightMargin">
                      <number>0</number>
                    </property>
                    <property name="bottomMargin">
                      <number>0</number>
                    </property>
                    <property name="horizontalSpacing">
                      <number>10</number>
                    </property>
                    <property name="verticalSpacing">
                      <number>5</number>
                    </property>
                    <!--================================================================-->
                    <item row="0" column="0">
                      <widget class="QLabel" name="label_size_att" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>Attribute:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="1" colspan="3">
                      <widget class="QComboBox" name="combosel_size_att">
                        <property name="sizeAdjustPolicy">
                          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                        </property>
                      </widget>
                    </item>
                    <!--================================================================-->
                    <item row="1" column="0">
                      <widget class="QLabel" name="label_limits_att" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>Limits:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1">
                      <widget class="QLineEdit" name="valuetext_size_vmin"/>
                    </item>
                    <item row="1" column="2">
                      <widget class="QToolButton" name="button_flip_size">
                        <property name="font">
                          <font>
                            <pointsize>12</pointsize>
                          </font>
                        </property>
                        <property name="styleSheet">
                          <string notr="true">padding: 0px</string>
                        </property>
                        <property name="text">
                          <string>â‡„</string>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="3">
                      <widget class="QLineEdit" name="valuetext_size_vmax"/>
                    </item>
                    <!--================================================================-->
                    <!-- <item row="2" column="0">
                      <widget class="QLabel" name="label_size_scaling" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>Scaling factor:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="2" column="1" colspan="3">
                      <widget class="QSlider" name="value_size_scaling">
                        <property name="maximum">
                          <number>11</number>
                        </property>
                        <property name="value">
                          <number>50</number>
                        </property>
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </widget>
                    </item> -->
                  </layout>
                </widget>
              </item>
              <!--================================================================-->
              <item row="2" column="0">
                <spacer name="verticalSpacer">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                    <size>
                      <width>20</width>
                      <height>40</height>
                    </size>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>
          <!--================================================================-->
          <!--============================ Color =============================-->
          <widget class="QWidget" name="color_tab">
            <attribute name="title">
              <string>Color</string>
            </attribute>
            <layout class="QGridLayout" name="color_grid">
              <property name="leftMargin">
                <number>4</number>
              </property>
              <property name="topMargin">
                <number>8</number>
              </property>
              <property name="rightMargin">
                <number>4</number>
              </property>
              <property name="bottomMargin">
                <number>4</number>
              </property>
              <property name="horizontalSpacing">
                <number>10</number>
              </property>
              <property name="verticalSpacing">
                <number>5</number>
              </property>
              <!--================================================================-->
              <item row="0" column="0">
                <widget class="QLabel" name="label_color_mode" alignment="Qt::AlignRight|Qt::AlignVCenter">
                  <property name="font">
                    <font>
                      <weight>50</weight>
                    </font>
                  </property>
                  <property name="text">
                    <string>Color mode:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QComboBox" name="combosel_color_mode">
                  <property name="maximumSize">
                    <size>
                      <width>80</width>
                      <height>16777215</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item row="0" column="2">
                <widget class="QColorBox" name="color_color">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                </widget>
              </item>
              <!--================================================================-->
              <item row="1" column="0" colspan="3">
                <widget class="QWidget" name="cmap_attributes">
                  <layout class="QGridLayout" name="cmap_grid">
                    <property name="leftMargin">
                      <number>0</number>
                    </property>
                    <property name="topMargin">
                      <number>0</number>
                    </property>
                    <property name="rightMargin">
                      <number>0</number>
                    </property>
                    <property name="bottomMargin">
                      <number>0</number>
                    </property>
                    <property name="horizontalSpacing">
                      <number>10</number>
                    </property>
                    <property name="verticalSpacing">
                      <number>5</number>
                    </property>
                    <!--================================================================-->
                    <item row="0" column="0">
                      <widget class="QLabel" name="label_cmap_attribute" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>Attribute:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="1" colspan="3">
                      <widget class="QComboBox" name="combosel_cmap_att">
                        <property name="sizeAdjustPolicy">
                          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                        </property>
                      </widget>
                    </item>
                    <!--================================================================-->
                    <item row="1" column="0">
                      <widget class="QLabel" name="label_cmap_limits" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>Limits:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1">
                      <widget class="QLineEdit" name="valuetext_cmap_vmin"/>
                    </item>
                    <item row="1" column="2">
                      <widget class="QToolButton" name="button_flip_cmap">
                        <property name="font">
                          <font>
                            <pointsize>12</pointsize>
                          </font>
                        </property>
                        <property name="styleSheet">
                          <string notr="true">padding: 0px</string>
                        </property>
                        <property name="text">
                          <string>â‡„</string>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="3">
                      <widget class="QLineEdit" name="valuetext_cmap_vmax"/>
                    </item>
                    <!--================================================================-->
                    <item row="2" column="0">
                      <widget class="QLabel" name="label_colormap" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>Colormap:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="2" column="1" colspan="3">
                      <widget class="QColormapCombo" name="combodata_cmap">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="sizeAdjustPolicy">
                          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                        </property>
                      </widget>
                    </item>
                    <!--================================================================-->
                    <item row="3" column="0">
                      <widget class="QLabel" name="label_cmap_nan_mode" alignment="Qt::AlignRight|Qt::AlignVCenter">
                        <property name="font">
                          <font>
                            <weight>50</weight>
                          </font>
                        </property>
                        <property name="text">
                          <string>NaN option:</string>
                        </property>
                      </widget>
                    </item>
                    <item row="3" column="1">
                      <widget class="QRadioButton" name="radio_cmap_nan_hide">
                        <property name="text">
                          <string>Hide</string>
                        </property>
                        <attribute name="nanCmapButtonGroup">
                          <string notr="true">nanCmapButtonGroup</string>
                        </attribute>
                      </widget>
                    </item>
                    <item row="3" column="2">
                      <widget class="QRadioButton" name="radio_cmap_nan_color">
                        <property name="text">
                          <string>Color</string>
                        </property>
                        <attribute name="nanCmapButtonGroup">
                          <string notr="true">nanCmapButtonGroup</string>
                        </attribute>
                      </widget>
                    </item>
                    <item row="3" column="3">
                      <spacer name="horizontalSpacer">
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                          <size>
                            <width>40</width>
                            <height>20</height>
                          </size>
                        </property>
                      </spacer>
                    </item>
                    <!--================================================================-->
                    <item row="4" column="1" colspan="3" name="cmap_nan_value_info">
                      <widget class="QStackedWidget" name="cmap_nan_value_stacked_widget">
                        <property name="currentIndex">
                          <number>0</number>
                        </property>
                        <widget class="QWidget" name="cmap_nan_info">
                          <layout class="QGridLayout" name="cmap_nan_info_grid">
                            <item row="0" column="0">
                              <widget class="QLabel" name="label_cmap_nan_info" alignment="Qt::AlignLeft">
                                <property name="wordWrap">
                                  <bool>true</bool>
                                </property>
                                <property name="font">
                                  <font>
                                    <weight>100</weight>
                                  </font>
                                </property>
                                <property name="text">
                                  <string>Points with NaN-values won't show in OpenSpace</string>
                                </property>
                                <property name="sizePolicy">
                                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                  </sizepolicy>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </widget>
                        <widget class="QWidget" name="cmap_nan_color">
                          <layout class="QGridLayout" name="cmap_nan_color_grid">
                            <item row="0" column="0">
                              <widget class="QLabel" name="label_cmap_nan_color" alignment="Qt::AlignLeft">
                                <property name="wordWrap">
                                  <bool>true</bool>
                                </property>
                                <property name="font">
                                  <font>
                                    <weight>50</weight>
                                  </font>
                                </property>
                                <property name="text">
                                  <string>NaN color:</string>
                                </property>
                              </widget>
                            </item>
                            <item row="0" column="1">
                              <widget class="QColorBox" name="color_cmap_nan_color">
                                <property name="sizePolicy">
                                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                  </sizepolicy>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </widget>
                      </widget>
                    </item>
                    <!--================================================================-->
                  </layout>
                </widget>
              </item>
              <!--================================================================-->
              <item row="2" column="0">
                <widget class="QLabel" name="label_7" alignment="Qt::AlignRight|Qt::AlignVCenter">
                  <property name="font">
                    <font>
                      <weight>50</weight>
                    </font>
                  </property>
                  <property name="text">
                    <string>Opacity:</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="1" colspan="2">
                <widget class="QLineEdit" name="valuetext_alpha"/>
              </item>
              <!--================================================================-->
              <item row="3" column="0">
                <spacer name="verticalSpacer_2">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                    <size>
                      <width>20</width>
                      <height>40</height>
                    </size>
                  </property>
                </spacer>
              </item>
              <item row="3" column="1" colspan="2">
                <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                    <size>
                      <width>40</width>
                      <height>1</height>
                    </size>
                  </property>
                </spacer>
              </item>
              <!--================================================================-->
            </layout>
          </widget>
          <!--================================================================-->
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>QColorBox</class>
      <extends>QLabel</extends>
      <header>glue.utils.qt.colors</header>
    </customwidget>
    <customwidget>
      <class>QColormapCombo</class>
      <extends>QComboBox</extends>
      <header>glue.utils.qt.colors</header>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>